doc/examples.md

```diff
@@ -120,15 +120,14 @@ In many cases it is useful to write files to the root filesystem. This example w
 The config makes use of the universally-defined "root" filesystem. This filesystem is defined within Ignition itself and roughly looks like the following. The "root" filesystem allows additional configs to reference the root filesystem, regardless of its type (e.g. btrfs, tmpfs, ext4).
 
 ```json ignition
-{
-  "ignition": { "version": "2.1.0" },
```

Comment:
Why is the version being removed in this example?

Reply:
Specifying the root filesystem generates a warning.  the `...` indicates a snippet and prevents the example from being validated.

Reply:
Oh, I missed the bit about snippets. In that case, looks good.